### PR TITLE
Test Idle PWM 

### PR DESCRIPTION
**Pull request by shadowm60**
Using existing hw jumpers we measure idle pin pwm.

Duty is not yet checked, but we do check if we have correct output frequency.

### CODE REVIEW

File: java_console/autotest/src/com/rusefi/PwmHardwareTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        /* save these for last to ensure logic is started */
+        ecu.sendCommand(CMD_LOGIC_PIN + " 0 PA5");
+        ecu.sendCommand(CMD_WRITECONFIG);
+        sleep(2 * Timeouts.SECOND);
+        ecu.sendCommand(getEnableCommand(Fields.CMD_SELF_STIMULATION));
+
+        sleep(2 * Timeouts.SECOND);
+
+        /* +-1% is still acceptable */
+        EcuTestHelper.assertEquals("Idle PWM freq", 160, SensorCentral.getInstance().getValue(Sensor.debugIntField1),0.01);
```
Comment (rusefillc) [inline java_console/autotest/src/com/rusefi/PwmHardwareTest.java:57]: 1% of 160 integer is "1"? that would probably be just fine here.

File: java_console/autotest/src/com/rusefi/PwmHardwareTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+        if (ControllerConnectorState.firmwareVersion == null)
```
Comment (rusefillc) [inline java_console/autotest/src/com/rusefi/PwmHardwareTest.java:60]: please remove this - this is not related to this test, one test having this is enough

File: java_console/autotest/src/com/rusefi/PwmHardwareTest.java (hunk 1 above)
Comment (shadowm60) [reply java_console/autotest/src/com/rusefi/PwmHardwareTest.java:57]: since it is and int at the end, yes

File: java_console/autotest/src/com/rusefi/PwmHardwareTest.java (hunk 2 above)
Comment (rusefillc) [reply java_console/autotest/src/com/rusefi/PwmHardwareTest.java:60]: https://github.com/rusefi/rusefi/commit/7492172722ccb3c0f3115eede34eaa169b647b40
